docs/variants/asus_kgpe_d16/setup.md

```diff
@@ -0,0 +1,214 @@
+ 1 (VCC)           | 2 (VCC_PROG)
+ 2 (GND)           | 7 (GND)
+ 3 (CS)            | 1 (CS_PROG#)
+ 4 (SCLK)          | 6 (CLK_PROG)
+ 5 (MISO)          | 8 (IO0_DI_PROG)
+ 6 (MOSI)          | 3 (IO1_DO_PROG)
+ 7 (NC)            | Not connected
+ 8 (NC)            | Not connected
+
+Additionally one has to tie the IO3_HOLD_PROG (qspimux pin 4) high, so connect
+it to the 3.3V permanently. One more additional GPIO is required in order to
+control the routing of SPI lines. The MUX_SEL (qsppimux pin 9) decides whether
+the programmer is allowed to access the SPI flash or the mainboard.
+
+The proposed connection used RTE GPIO400 to control MUX_SEL and RTE GPIO401 to
+control IO3_HOLD_PROG. Simple flashing script looks as follows:
+
+```
+# select flash <-> programmer
+echo "0" > /sys/class/gpio/gpio400/value
+# deassert SPI HOLD#
+echo "1" > /sys/class/gpio/gpio401/value
+
+flashrom -w firmware.bin -p linux_spi:dev=/dev/spidev1.0,spispeed=16000
+
+# select flash <-> board
+echo "1" > /sys/class/gpio/gpio400/value
+```
+
+ RTE header J10 pin | qspimux J101 pin
+:------------------:|:-------------------:
+ 1 (GPIO400)        | 9 (MUX_SEL)
+ 2 (GPIO401)        | 4 (IO3_HOLD_PROG)
+
+![](/images/qspimux_pin_header.png)
+
+![](/images/qspimux_kgpe.jpeg)
+
+> qspimux schematics: https://github.com/felixheld/qspimux/blob/master/qspimux/qspimux.pdf
+
+#### Serial
+
+Asus KGPE-D16 has a DB9 serial connector for debug UART. Connect it to the RTE
+via RS232 DB9 cable.
+
+![](/images/kgpe_rear_panel.png)
+
+#### TPM
+
+Asus KGPE-D16 has a LPC header for TPM or debugging. Since there is no other
+option to connect a TPM, we use the header for that purpose.
+
+Example connection of PC Engines TPM 20pin module (tpm1a) to TPM header on
+KGPE-D16:
+
+![](/images/kgpe_tpm_header.png)
+
+| tpm1a         | Asus KGPE-D16        |
+|:-------------:|:--------------------:|
+| PCLK pin 1    | TPM pin 1  (PCICLK)  |
+| LAD0 pin 3    | TPM pin 11 (LAD0)    |
+| LAD1 pin 5    | TPM pin 10 (LAD1)    |
+| LAD2 pin 7    | TPM pin 8  (LAD2)    |
+| LAD3 pin 9    | TPM pin 7  (LAD3)    |
+| FRAME# pin 11 | TPM pin 3  (FRAME)   |
+| RESET# pin 13 | TPM pin 5  (PCIRST#) |
+| GND pin 17    | TPM pin 2  (GND)     |
+| V3 pin 18     | TPM pin 9  (+3V)     |
+| SERIRQ pin 19 | TPM pin 16 (SERIRQ)  |
+
+> For PC Engines TPM pinout refer to https://www.pcengines.ch/schema/tpm1a.pdf
+
+#### Power management
+
+Asus KGPE-D16 provides a front panel pin header (PANEL1):
+
+![](/images/kgpe_front_panel.png)
+
+| RTE header J11 pin     | Asus KGPE-D16             |
+|:----------------------:|:-------------------------:|
+| 8 (OC buffer output)   | PANEL1 pin 11 (PWR)       |
+| 9 (OC buffer output)   | PANEL1 pin 17 (RESET)     |
+
+Additionally connect one of the PANEL1 ground pins to one of the ground pins on
+RTE.
+
+### MANUAL
+
+For more details about KGPE-D16 refer to the manual:
+https://dlcdnets.asus.com/pub/ASUS/mb/SocketG34(1944)/KGPE-D16/Menual_QVL/E8847_KGPE-D16.pdf
```

Comment:
@miczyg1 maybe we should drop that to this repo? I'm not sure how long this manual will be available on ASUS website

Reply:
done